kong/init.lua

```diff
@@ -1375,6 +1376,26 @@ function Kong.serve_cluster_listener(options)
 end
 
 
+local stream_plugins_api_loaded = false
+function Kong.stream_api()
+  if not stream_plugins_api_loaded then
+    local utils = require "kong.tools.utils"
+
+    for k in pairs(kong.configuration.loaded_plugins) do
+      local loaded, custom_endpoints = utils.load_module_if_exists("kong.plugins." .. k .. ".api")
+      if loaded and type(custom_endpoints._stream) == "table" then
+        ngx.log(ngx.DEBUG, "---------------------------- have stream --------------")
```

Comment:
Maybe include the plugin's name in the debug log as well and change the format to something that is similar to others (e.g. without the dashes).

Reply:
done!  (debugging leftovers....)